pkg/cloudprovider/aws/testdata/br_userdata_merged.golden

```diff
@@ -0,0 +1,15 @@
+[settings]
+[settings.kubernetes]
+api-server = 'https://test-cluster'
+cluster-certificate = 'ca-bundle'
+cluster-name = 'test-cluster'
+max-pods = 110
+
+[settings.kubernetes.node-labels]
+'karpenter.sh/capacity-type' = 'on-demand'
+'karpenter.sh/provisioner-name' = '%s'
+
+[settings.kubernetes.node-taints]
+
+[settings.kubernetes.eviction-hard]
+'memory.available' = '12%%'
```

Comment:
Where does the double percent sign come from?

Reply:
I have to format this file at runtime to inject the provisioner-name on #L10 of this file. `%%` was to escape